chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,863 @@
+    }
+
+    return url_in + add + var_in;
+  }
+
+  // Callback methods for replacer
+  replacer_onEach(text, re, new_text, value, key) {
+    const regex = new RegExp(`%${this.escapeRegExp(key)}%`, 'gi');
+    const replaced = text.replace(regex, value);
+    return replaced;
+  }
+
```

Biome (1.9.4)

[error] Duplicate class member name "replacer_onEach"



(lint/suspicious/noDuplicateClassMembers)